mypyc/irbuild/expression.py

```diff
@@ -581,16 +584,16 @@ def translate_str_format_percent_sign(builder: IRBuilder,
         elif isinstance(rhs, Expression):
             exprs.append(rhs)
 
-        substitutions = convert_expr(builder, format_ops, exprs, format_expr.line)
-        if substitutions is not None:
-            return join_formatted_strings(builder, literals, substitutions, format_expr.line)
+        if isinstance(format_expr, BytesExpr):
+            substitutions = convert_expr_to_bytes(builder, format_ops, exprs, format_expr.line)
```

Comment:
Rename the function to `convert_format_expr_to_bytes`, for example, for clarity?